### PR TITLE
update kubectl output in examples/glusterfs

### PR DESCRIPTION
**Pull request by caesarxuchao**
ref #10168
@JanetKuo @krousey @nikhiljindal @bgrant0607

### CODE REVIEW

File: examples/glusterfs/README.md [hunk 1]
```diff
@@ -67,13 +67,14 @@ You can verify that the pod is running:
 
 ```shell
 $ kubectl get pods
-POD         IP            CONTAINER(S)   IMAGE(S)              HOST                                  LABELS    STATUS    CREATED          MESSAGE
-glusterfs   10.244.2.13                                        kubernetes-minion-151f/23.236.54.97   <none>    Running   About a minute   
-                          glusterfs      kubernetes/pause                                                      Running   About a minute   
+NAME             READY     STATUS    RESTARTS   AGE
+glusterfs        1/1       Running   0          3m
 
+$ kubectl get pods glusterfs -t {{.status.hostIP}}
```
Comment (JanetKuo) [inline examples/glusterfs/README.md]: When I try this command, linebreak (\n) is missing so it looks weird to me. 

`$ cluster/kubectl.sh get pods redis-master-lc771 -t {{.status.hostIP}}`
`10.240.163.41$ `

**Comment by nikhiljindal** [comment]
LGTM

File: examples/glusterfs/README.md (hunk 1 above)
Comment (caesarxuchao) [inline examples/glusterfs/README.md]: I agree. @nikhiljindal, do you know if this is on purpose?

**Comment by yujuhong** [comment]
@nikhiljindal, is this ok to merge?

File: examples/glusterfs/README.md (hunk 1 above)
Comment (nikhiljindal) [inline examples/glusterfs/README.md]: Not on purpose. I agree adding /n is better.
@caesarxuchao Please add a /n.

Comment (caesarxuchao) [inline examples/glusterfs/README.md]: I address the comments. I believe there are other similar cases in examples/, I will fix them when I have a chance.

Comment (nikhiljindal) [inline examples/glusterfs/README.md]: Thanks!

**Pull request merged**